rust-src/aggregate_sig/src/ffi.rs

```diff
@@ -224,6 +299,53 @@ mod test {
         }
     }
 
+    // Verify that an aggregate signature created from two message groups each
+    // signed by two keys can be succesfully verified.
+    #[test]
+    fn test_verify_aggregate_hybrid_ffi() {
+        let mut rng: StdRng = SeedableRng::from_rng(thread_rng()).unwrap();
```

Comment:
```suggestion
        let mut rng = thread_rng();
```
I don't see any reason for the complex seeding that is done here now.

Reply:
I think it was due to a copy/paste. I've also simplified above.